docs/editors/emacs.md

```diff
@@ -28,10 +28,17 @@ Next, update your Emacs configuration to load `lsp-scala` along with its
 dependencies
 
 ```el
+;; Add melpa-stable to your packages repositories
+(add-to-list 'package-archives '("melpa-stable" . "http://stable.melpa.org/packages/") t)
```

Comment:
One more thing: any reason not to use https here?

Reply:
Not really, I just used the one in my `.emacs.d`. I'm gonna change it.